Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.4
+
+# cache-fill before COPY
+# (docker/docker via git clone --depth=1 for speed)
+RUN mkdir -p /go/src/github.com/docker \
+	&& git clone --depth=1 https://github.com/docker/docker.git /go/src/github.com/docker/docker
+ENV GOPATH $GOPATH:/go/src/github.com/docker/docker/vendor
+# (also, naughty vbatts using things outside pkg/ from docker/docker)
```

Comment:
i know. and it's bitten me too. perhaps we'll switch it to the docker/distribution soon